### PR TITLE
feature: moderator symbol

### PR DESCRIPTION
**Pull request by ebarthur**

### CODE REVIEW

File: client/app/components/username.tsx [hunk 1]
```diff
@@ -17,6 +17,10 @@ function Username({ showVerfied, user }: Props) {
 			{user.username === "notgr" && (
 				<div className="inline-block i-lucide-crown text-amber-500" />
 			)}
+			{/* This should be for moderators */}
+			{user.role === "moderator" && user.username !== "notgr" && (
+				<div className="inline-block i-lucide-gem text-amber-500" />
```
Comment (blackmann) [inline client/app/components/username.tsx]: hmm gem? 

what about ghost or zap.

Comment (blackmann) [reply client/app/components/username.tsx]: And let's use a different color.

Comment (ebarthur) [reply client/app/components/username.tsx]: I'm going with the zap in red-500. The lucide ghost just doesn't bang

Check it out
imageimage

Comment (ebarthur) [reply client/app/components/username.tsx]: Lemme know what you think? Then I'll commit

Comment (blackmann) [reply client/app/components/username.tsx]: Looks cool... I like this too

File: client/app/components/username.tsx [hunk 2]
```diff
@@ -17,6 +17,10 @@ function Username({ showVerfied, user }: Props) {
 			{user.username === "notgr" && (
 				<div className="inline-block i-lucide-crown text-amber-500" />
 			)}
+			{/* This should be for moderators */}
+			{user.role === "moderator" && user.username !== "notgr" && (
```
Comment (blackmann) [inline client/app/components/username.tsx]: oh I'm moderator... I want some of the zap

Comment (ebarthur) [reply client/app/components/username.tsx]: So you get a crown and a zap? I'm not sure you'll like the color combo tho. You suggest the change😅

**Review by blackmann — APPROVED** [review]

**Pull request merged**